concepts/strings-package/about.md

```diff
@@ -1,5 +1,17 @@
 # About
 
-## Source code encoding
+The [`strings` package](https://pkg.go.dev/strings) contains many useful functions to work on strings:
 
-Source code files in Go are always `UTF-8` encoded. This makes it very easy to use special characters in strings.
+| Function                                      | Purpose                                                             |
+| --------------------------------------------- | ------------------------------------------------------------------- |
+| [ToLower](https://pkg.go.dev/strings#ToLower) | Convert the string to lower case                                    |
```

Comment:
I have no idea what the most commonly used string functions are in Go. Please feel free to suggest any.

Reply:
`ToUpper`, `HasPrefix`, `HasSuffix`, `Replace`, `TrimSpace`, `Split`

Reply:
Thanks